recipes/libcurand/meta.yaml

```diff
@@ -0,0 +1,145 @@
+        - test -L $PREFIX/lib/libcurand.so.{{ version.split(".")[0] }}                            # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libcurand.so.{{ version }}                # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libcurand.so.{{ version.split(".")[0] }}  # [linux]
+        - if not exist %LIBRARY_BIN%\curand64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+    about:
+      home: https://developer.nvidia.com/curand
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: cuRAND native runtime libraries
+      description: |
+        cuRAND native runtime libraries
+      doc_url: https://docs.nvidia.com/cuda/curand/
+
+  - name: libcurand-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("libcurand", max_pin="x") }}
+    files:
+      - lib/libcurand.so                            # [linux]
+      - lib/pkgconfig                               # [linux]
+      - targets/{{ target_name }}/include           # [linux]
+      - targets/{{ target_name }}/lib/libcurand.so  # [linux]
+      - targets/{{ target_name }}/lib/stubs         # [linux]
+      - targets/{{ target_name }}/libcurand/src     # [linux and not aarch64]
+      - Library\include                             # [win]
+      - Library\lib                                 # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("libcurand", exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - libcublas-static >={{ version }}      # [linux]
+    test:
+       commands:
+         - test -L $PREFIX/lib/libcurand.so                                    # [linux]
+         - test -f $PREFIX/targets/{{ target_name }}/include/curand.h          # [linux]
+         - test -L $PREFIX/targets/{{ target_name }}/lib/libcurand.so          # [linux]
+         - test -f $PREFIX/targets/{{ target_name }}/lib/stubs/libcurand.so    # [linux]
+         - if not exist %LIBRARY_INC%\curand.h exit 1                          # [win]
+         - if not exist %LIBRARY_LIB%\curand.lib exit 1                        # [win]
+    about:
+      home: https://developer.nvidia.com/curand
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: cuRAND native runtime libraries
+      description: |
+        cuRAND native runtime libraries
+      doc_url: https://docs.nvidia.com/cuda/curand/
+
+  - name: libcurand-static
```

Comment:
The `about` section is missing in this package.

Reply:
Added.